src/org/javarosa/core/model/FormDef.java

```diff
@@ -1525,6 +1528,12 @@ private void attachControlsToInstanceData(TreeElement node) {
         }
     }
 
+    /**
+     * Returns the given question definition related to a form node reference.
+     * @param ref Reference to a form node.
```

Comment:
Looking here, https://www.oracle.com/technetwork/java/javase/documentation/index-137868.html, I see that `@param` descriptions start with lower case and don’t have periods if there’s just one phrase.

Reply:
Thanks! Will update

Reply:
Corrected this one too. Marking resolved.